maskrcnn_benchmark/modeling/detector/generalized_rcnn.py

```diff
@@ -49,10 +49,9 @@ def forward(self, images, targets=None):
         features = self.backbone(images.tensors)
         proposals, proposal_losses = self.rpn(images, features, targets)
         if self.roi_heads:
-            x, result, detector_losses = self.roi_heads(features, proposals, targets)
+            result, detector_losses = self.roi_heads(features, proposals, targets)
```

Comment:
I'd rather have this as a potentially separate PR, as this needs further discussion. can you revert this?